### ISSUE TITLE
Build 519 can't create pocketmine.yml

### ISSUE THREAD

**Issue by robske110** [ImagicalCorp/ImagicalMine#669]
lol it tries to access it even if it is not there!
```YEAH THIS IS FUCKING PHP
[13:11:41] system> -------------------------------------------------------------------------------------------
[13:11:41] system> |  _                       _           _ __  __ _                                         |
[13:11:41] system> | (_)                     (_)         | |  \/  (_)                                        |
[13:11:41] system> |  _ _ __ ___   __ _  __ _ _  ___ __ _| | \  / |_ _ __   ___                              |
[13:11:41] system> | | | '_ ` _ \ / _` |/ _` | |/ __/ _` | | |\/| | | '_ \ / _ \      ImagicalMine 1.0dev    |
[13:11:41] system> | | | | | | | | (_| | (_| | | (_| (_| | | |  | | | | | |  __/                             |
[13:11:41] system> | |_|_| |_| |_|\__,_|\__, |_|\___\__,_|_|_|  |_|_|_| |_|\___|    for MCPE v0.13.1 alpha   |
[13:11:41] system> |                     __/ |                                                               |
[13:11:41] system> |                    |___/                                                                |
[13:11:41] system> ------------------------------------------------------------------------------------------- 
[13:11:41] system>                                                                                                                         
[13:11:41] system> ImagicalMine is a third-party build of PocketMine-MP, distributed under the LGPL licence
```
Fatal error: Uncaught exception 'RuntimeException' with message 'file_get_contents(/Users/Tim/Desktop/Pocket_Server_0.13.x/src/pocketmine/resources/pocketmine.yml): failed to open stream: No such file or directory' in phar:///Users/Tim/Desktop/Pocket_Server_0.13.x/PocketMine-MP.phar/src/pocketmine/katana/Katana.php:101
Stack trace:
´´´
#0 [internal function]: ExceptionHandler::handler(2, 'file_get_conten...', 'phar:///Users/T...', 101, Array)
#1 phar:///Users/Tim/Desktop/Pocket_Server_0.13.x/PocketMine-MP.phar/src/pocketmine/katana/Katana.php(101): file_get_contents('/Users/Tim/Desk...')
#2 phar:///Users/Tim/Desktop/Pocket_Server_0.13.x/PocketMine-MP.phar/src/pocketmine/katana/Katana.php(79): pocketmine\katana\Katana->initConfig()
```
#3 phar:///Users/Tim/Desktop/Pocket_Server_0.13.x/PocketMine-
MP.phar/src/pocketmine/Server.php(1577): pocketmine\katana\Katana->__construct(Object(pocketmine\Server))
´´´
```php
#4 phar:///Users/Tim/Desktop/Pocket_Server_0.13.x/PocketMine-MP.phar/src/pocketmine/PocketMine.php(469): pocketmine\Server->__constru in phar:///Users/Tim/Desktop/Pocket_Server_0.13.x/PocketMine-MP.phar/src/pocketmine/katana/Katana.php on line 101
´0`
^Clogout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
```
´´´

**Comment by robske110** [comment]
Github scrapped the coding style

**Comment by thebigsmileXD** [comment]
Not github. You did. (the devs)

**Comment by robske110** [comment]
I am referring to the crash dump style

**Comment by MCDad** [comment]
This is the same thing I reported this morning in #662.  Still not fixed in Jenkins #519

**Comment by keithkfng** [comment]
Try the latest build - Katana.php shouldn't be finding the files now. It's usually found in Server.php but I was lazy getting rid of imagicalmine.yml XD
